src/main/java/bio/terra/cli/app/utils/AppDefaultCredentialUtils.java

```diff
@@ -95,7 +83,9 @@ public static GoogleCredentials getApplicationDefaultCredentials() {
     try {
       return GoogleCredentials.getApplicationDefault();
     } catch (IOException ioEx) {
-      throw new UserActionableException("Application default credentials are not defined.", ioEx);
+      throw new UserActionableException(
+          "Application default credentials are not defined. Run `gcloud auth application-default login`",
```

Comment:
That advice works for local mode, but what if we're on a notebook and somehow can't get the ADCs?

Reply:
GCP Notebook should always have ADC.